book/src/feature/lang_opaque.md

```diff
@@ -1,18 +1,18 @@
 # Arbitrary types (opaque)
 
-Any Rust type, even if it is not supported using features of this library, can be used in Dart. This is done by wrapping it with `Opaque`.
+Any Rust type, even if it is not supported using features of this library, can be used in Dart. This is done by wrapping it with `RustOpaque` also Rust can get opaque Dart type in the `DartOpaque` wrap.
 
-The Dart opaque objects should be disposed *manually*, though it will also be disposed when it is GCed, that is discouraged, due to [suggestions by Dart team](https://github.com/fzyzcjy/flutter_rust_bridge/issues/775#issuecomment-1274635037). Think of it just like a lot of Flutter objects that we are familiar with, such as `ui.Image` - we have to manually dispose them as well.
+The Dart rust opaque objects should be disposed *manually*, though it will also be disposed when it is GCed, that is discouraged, due to [suggestions by Dart team](https://github.com/fzyzcjy/flutter_rust_bridge/issues/775#issuecomment-1274635037). Think of it just like a lot of Flutter objects that we are familiar with, such as `ui.Image` - we have to manually dispose them as well. The Dart opaque objects are always deleted on the parent Dart thread.
```

Comment:
Hmm what is "dart rust opaque" that should be disposed manually? Looks like a typo